_templates/sidebar.html

```diff
@@ -3,15 +3,24 @@
     <head>
       <title>HJD</title>
 	</head>
```

Comment:
you do not need this in the sidebar, beause the sidebar is only one piece fo the whole page